site/app/models/Gradeable.php

```diff
@@ -887,4 +891,8 @@ public function saveData() {
     public function getSyllabusBucket() {
         return $this->bucket;
     }
+    
+    public function isPeer() {
+        return $this->peer_grading;
+    }
```

Comment:
this function would make more sense named "isPeerGrading" (which then also makes it a magic function so it can just be a `@method`).